pwndbg.nix

```diff
@@ -0,0 +1,73 @@
+{
+  pkgs ? import <nixpkgs> { }
+  , python3 ? pkgs.python3
+  , gdb ? pkgs.gdb
+}:
+let
+  binPath = pkgs.lib.makeBinPath ([
+    python3.pkgs.pwntools   # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/wrappers/checksec.py#L8
+  ] ++ pkgs.lib.optionals pkgs.stdenv.isLinux [
+    python3.pkgs.ropper     # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/ropper.py#L30
+    python3.pkgs.ropgadget  # ref: https://github.com/pwndbg/pwndbg/blob/2022.12.19/pwndbg/commands/rop.py#L32
```

Comment:
The '# ref' link links to 2022 version, can we update those?

Reply:
done